plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -435,7 +435,7 @@ function MenuDialog:onConfigChoose(values, name, event, args, position)
                 self:onChangeContextStatus(args, position)
             end
         end
-        UIManager:setDirty(nil, "ui", nil, true)
```

Comment:
@weijiuqiao: Any idea why this was explicitly requesting dithering?

Reply:
My mistake, it’s not needed. There’s no image involved.

Reply:
Cool, thanks for the confirmation ;).